mpd/pssh.go

```diff
@@ -0,0 +1,41 @@
+package mpd
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+)
+
+func makePSSHBox(systemID, payload []byte) ([]byte, error) {
+	if len(systemID) != 16 {
+		return nil, fmt.Errorf("SystemID must be 16 bytes, was: %d", len(systemID))
+	}
+
+	psshBuf := &bytes.Buffer{}
+	size := uint32(12 + 16 + 4 + len(payload)) // 3 uint32s, systemID, "pssh" string and payload
```

Comment:
NIT: Order in comment is wrong, should be `// size, "pssh" string, null, systemID, payload`